### ISSUE TITLE
Training dataset question

### ISSUE THREAD

**Issue by Luciennnnnnn** [IceClear/StableSR#124]
May I ask, for the DF2K and DIV8K datasets, did you crop them into small patches in advance during training? Because this will affect the proportion of various types of data in the final training set. For example, if cropped in advance, there will be many patches from DF2K and DIV8K in the training set.

**Comment by Luciennnnnnn** [comment]
@IceClear cc.

**Comment by IceClear** [comment]
We follow the same settings as RealESRGAN. We did not process the data in advance.

**Comment by Luciennnnnnn** [comment]
> We follow the same settings as RealESRGAN. We did not process the data in advance.

Hi, this is still a bit confusing. You mentioned following RealESRGAN, but RealESRGAN by default crops the images in DF2K into small patches. To clarify, did you not preprocess the image and read it as a complete image?

**Comment by IceClear** [comment]
> > We follow the same settings as RealESRGAN. We did not process the data in advance.
> 
> Hi, this is still a bit confusing. You mentioned following RealESRGAN, but RealESRGAN by default crops the images in DF2K into small patches. To clarify, did you not preprocess the image and read it as a complete image?

Yes. Pre-cropping as RealESRGAN and directly reading the whole image are not very different from my view. Finally, the GT should be a fixed resolution and cropping is a necessary step. For details, you can refer to the related code of this repo.
